test/System.Web.Http.Owin.Test/OwinHttpRequestContextTests.cs

```diff
@@ -564,6 +564,30 @@ public void VirtualPathRootGet_ReturnsContextRequestPathBase()
             }
         }
 
+        // Regression test for #203.
+        [Fact]
+        public void VirtualPathRootGet_ReturnsUnescapedContextRequestPathBase()
+        {
+            // Arrange
+            var expectedVirtualPathRoot = "/a b";
+            var owinRequestMock = new Mock<IOwinRequest>(MockBehavior.Strict);
+            owinRequestMock
+                .Setup(r => r.PathBase)
+                .Returns(new PathString(expectedVirtualPathRoot));
```

Comment:
This test is a `[Fact]` because `PathString` throws if virtual path root is escaped or doesn't start with `'/'`.